modules/bitcoind.nix

```diff
@@ -222,6 +222,7 @@ in {
         User = "${cfg.user}";
         Group = "${cfg.group}";
         ExecStart = "${cfg.package}/bin/bitcoind ${cmdlineOptions}";
+        ExecStop = "/nix/store/0cvw3jxxncnm3whckl0qi72bs4g35833-coreutils-8.30/bin/rm -f ${cfg.dataDir}/zmqpubrawblock.sock ${cfg.dataDir}/zmqpubrawtx.sock";
```

Comment:
You cannot assume that coreutils are always at this location. Use `${pkgs.coreutils}/bin/rm` instead.